recipes/ckanbatch-deploy.rb

```diff
@@ -156,3 +151,8 @@
     owner "root"
     group "root"
 end
+
+service "supervisord restart" do
+    service_name "supervisord"
+    action [:stop, :start]
```

Comment:
we kind of use this incorrectly in chef, its kind of meant to be attached as a pre/post hook on the element needing the restart instead of forcing the service to change 'state'

Reply:
Subscriptions like those don't seem well suited to monitoring *multiple* files...